src/main/java/ch/njol/skript/effects/EffExit.java

```diff
@@ -104,7 +103,7 @@ private static int numLevels(int type) {
 			return currentSections.size();
 		int r = 0;
 		for (TriggerSection s : currentSections) {
-			if (type == CONDITIONALS ? s instanceof SecConditional : s instanceof SecLoop || s instanceof SecWhile)
+			if (type == CONDITIONALS ? s instanceof SecConditional : s instanceof LoopSection)
```

Comment:
Now https://github.com/SkriptLang/Skript/pull/4839 is merged, can you edit EffContinue as well?

Reply:
Sure :)